### ISSUE TITLE
Android : Build failed in react-native@0.47.0-rc.1

### ISSUE THREAD

**Issue by ayushnawani** [react-native-community/react-native-svg#397]
**Description** :

I am getting the following error on android.

```
:react-native-svg:compileReleaseJavaWithJavac
:react-native-svg:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).

/Users/ayushnawani/Project/SvgTest/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/SvgPackage.java:47: error: method does not override or implement a method from a supertype
    @Override
    ^
1 error

:react-native-svg:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-svg:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 13.679 secs

```

**Dependencies** : 

```
		"react": "16.0.0-alpha.12",
		"react-native": "0.47.0-rc.1",
		"react-native-svg": "5.3.0"
```

**Suggested Solution** : 

In react-native@0.47.0-rc.1,  **ReactPackager** interface is updated and it does not contain the following method 
```
public List<Class<? extends JavaScriptModule>> createJSModules() 
```
i  fixed this error in my project by removing **createJSModules** method from **SvgPackage.java** 

**Sent Pull Request** : #398

**Comment by magicismight** [comment]
https://github.com/react-native-community/react-native-svg/tree/5.4.0

**Comment by ttdat89** [comment]
I think that you should merge now because this issue also happened in react-native@0.46.4

**Comment by magicismight** [comment]
@ttdat89 I have tested `react-native-svg@5.3.0` on 0.46.4.
It works fine.

Try to print the version information.
```
npm ls react-native
npm ls react-native-svg
```

**Comment by doomsower** [comment]
I think it landed too soon, because with `react-native@0.46.4` and `react-native-svg@5.3.1` the override is missing and causes this:
```
:react-native-svg:compileReleaseJavaWithJavac
:react-native-svg:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
/path/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/SvgPackage.java:23: error: SvgPackage is not abstract and does not override abstract method createJSModules() in ReactPackage
public class SvgPackage implements ReactPackage {
       ^

```
Had to revert to `5.3.0`

**Comment by reza7rm** [comment]
I still have the issue with react-native-svg@5.3.3

**Comment by msand** [comment]
@reza7rm try this: https://github.com/react-native-community/react-native-svg/pull/430
npm i -s react-native-svg@https://github.com/msand/react-native-svg.git#13d5157

**Comment by reza7rm** [comment]
@msand Thanks,I donno why but actually it was solved using yarn!

**Comment by msand** [comment]
Alright, great. Btw, the pull request/fork contains lots of svg spec conformance improvements, for images, transforms, viewbox, gradients, text, text on a path etc. which will be released in the next major version: 6.0.0
